dvc/render/converter/vega.py

```diff
@@ -241,6 +241,14 @@ def flat_datapoints(self, revision):  # noqa: C901, PLR0912
         else:
             common_prefix_len = 0
 
+        props_update["anchors_y_defn"] = [
+            {
+                FILENAME_FIELD: _get_short_y_file(y_file, common_prefix_len),
+                FIELD_FIELD: y_field,
+            }
+            for y_file, y_field in ys
```

Comment:
[F] Going to send this as JSON to the extension so can't leave it as a list of tuples.

Reply:
might not need to do this